platform/migration/platform-migration-utils/src/main/java/org/codice/ddf/configuration/migration/util/VersionUtils.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.configuration.migration.util;
+
+import static java.lang.Float.parseFloat;
+import static org.codice.ddf.migration.Migratable.IMPORT_UNSUPPORTED_MIGRATABLE_VERSION_ERROR;
+
+import org.codice.ddf.migration.ImportMigrationContext;
+import org.codice.ddf.migration.MigrationException;
+
+/** Utility class to help with migratable version checking */
+public class VersionUtils {
+
+  /** Prevents instantiation. */
+  private VersionUtils() {}
+
+  /**
+   * Will compare the exported migratable version with this system's migratable version.
+   *
+   * @param context The import migration context
+   * @param exportedVersion the migratable version from the export
+   * @param currentVersion the migratable version for this system
+   * @param migratableId the migratable id
+   * @return <code>false</code> if the exported version is greater than the current version or if
+   *     either of the versions cannot be parsed to a float. Otherwise, return <code>true</code>.
+   */
+  public static boolean isValidMigratableVersion(
+      ImportMigrationContext context,
+      String exportedVersion,
+      String currentVersion,
+      String migratableId) {
+    boolean isValidMigratableVersion;
+    try {
```

Comment:
❗️ Migratable versions were never set to be floats. They could be any strings that made sense to the migratable. I would suggest we first use `equals()` and then go with the parsing as float.

Reply:
I added a new method to this util class for verifying the validity of the versions, assuming the versions are floats.